perllib/FixMyStreet/Cobrand/Peterborough.pm

```diff
@@ -899,6 +899,31 @@ sub waste_munge_request_data {
     $data->{category} = $self->body->contacts->find({ email => "Bartec-$id" })->category;
 }
 
+sub waste_munge_bulky_data {
+    my ($self, $data) = @_;
+
+    my $c = $self->{c};
+
+    $data->{title} = "Bulky goods collection";
+    $data->{detail} = "Address: " . $c->stash->{property}->{address};
+    $data->{extra_DATE} = $data->{chosen_date};
+
+    # XXX loop here, plus might be more than 5 in future
+    $data->{extra_ITEM_01} = $data->{item1};
```

Comment:
Should it be e.g. `item_1.item` (i.e. matching what's set up in the form like in https://github.com/mysociety/fixmystreet/pull/4092/files#diff-d8cccc8a7adb37877cb4c6e1bb46ca3eddbc410055421d1b717d9c833cea163e)?

Reply:
I think it's best to keep it consistent with what's already on the branch rather than introducing a dependency on another branch. When this PR or the other gets merged then we can update this to handle the correct data structure.

Reply:
FYI I am merging https://github.com/mysociety/fixmystreet/pull/4092 onto staging, and I have changed this portion to
```
    $data->{extra_ITEM_01} = $data->{item_1}{item};
    $data->{extra_ITEM_02} = $data->{item_2}{item};
    $data->{extra_ITEM_03} = $data->{item_3}{item};
    $data->{extra_ITEM_04} = $data->{item_4}{item};
    $data->{extra_ITEM_05} = $data->{item_5}{item};
```
in order to make things work.